build/build_edge.sh

```diff
@@ -234,6 +235,16 @@ if [[ $full == 1 ]]; then
   sed -i 's/^CONFIG_packagegroup-petalinux-jupyter.*//g' project-spec/configs/rootfs_config
   sed -i 's/^CONFIG_kernel-devsrc.*//g' project-spec/configs/rootfs_config
   sed -i 's/^CONFIG_xrt-dev.*//g' project-spec/configs/rootfs_config
+  if ! grep -Fxq "CONFIG_libdw=y" project-spec/configs/rootfs_config; then
+    echo CONFIG_libdw=y >> project-spec/configs/rootfs_config
```

Comment:
we need to add this in xrt-recipe instead of adding in this script
https://gitenterprise.xilinx.com/Yocto/meta-xilinx/blob/rel-v2021.2/meta-xilinx-bsp/recipes-xrt/xrt/xrt_git.bb#L21